pkg/kubelet/apis/config/types.go

```diff
@@ -355,6 +355,9 @@ type KubeletConfiguration struct {
 	// The purpose of this format is make sure you have the opportunity to notice if the next release hides additional metrics,
 	// rather than being surprised when they are permanently removed in the release after that.
 	ShowHiddenMetricsForVersion string
+	// DisableAcceleratorUsageMetrics disables the accelerator metrics collected by cadvisor.
+	// In the future this option will default to true.
```

Comment:
nit: if in future the default will be that metrics are disabled, it will be more logical to call a setting `EnableAcceleratorUsageMetrics`. This way when you intent to enable metrics, you look for `Enable` flag, not unsetting the `Disabled` flag

Reply:
I was thinking about that, I'm pretty much in favor of having "Enable" rather than disable, I'll update the PR later today!
Thanks for the feedback!

Reply:
Updated!